automatic_gadget_generator/generator.html

```diff
@@ -2,11 +2,11 @@
 <head>
```

Comment:
I guess for this, the comments are the same as above. make sure the two copies are consistent.